usaspending_api/core/validator/tinyshield.py

```diff
@@ -302,3 +358,15 @@ def recurse_append(self, struct, mydict, data):
             else:
                 mydict[level] = {}
                 self.recurse_append(struct, mydict[level], data)
+
+    @staticmethod
+    def get_model_by_name(models, name):
```

Comment:
Handy function. I'm unsure I would include it in the TinyShield class since it is a utility

Reply:
You thinking a standalone function in the TinyShield file or would you want that in a separate file?

Reply:
We can place it in `utils.py` in the same directory